### PR TITLE
[docathon] vale clean-up pass #1

### PR DESCRIPTION
**Pull request by cmpadden**
## Summary & Motivation

- Fixes linting errors thrown by Vale (36 errors remain, but many are TODOs or false-positives)

## How I Tested These Changes

- Running `vale` locally

## Changelog [New | Bug | Docs]

NOCHANGELOG

### CODE REVIEW

File: docs/docs-beta/docs/dagster-plus/access/authentication/azure-ad-sso.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ dagster-cloud organization settings saml upload-identity-provider-metadata <path
    --url https://<organization_name>.dagster.cloud
 ```
 
-## Step 4: Create a test user \{#test-user}
+## Step 4: create a test user \{#test-user}
```
Comment (cmpadden) [inline docs/docs-beta/docs/dagster-plus/access/authentication/azure-ad-sso.md:78]: @erinkcochran87 I found we can support uppercase following `:` with the `indicators` parameter in our style -- I made these changes before updating the rule. What is our stance on this?

File: docs/vale/styles/Dagster/acronyms.yml [hunk 2]
```diff
@@ -8,29 +8,43 @@ first: '\b([A-Z]{3,5})\b'
 second: '(?:\b[A-Z][a-z]+ )+\(([A-Z]{3,5})\)'
 # ... with the exception of these:
 exceptions:
-  - API
+  - ACR
```
Comment (cmpadden) [inline docs/vale/styles/Dagster/acronyms.yml:11]: Alphabetized

File: docs/vale/styles/config/vocabularies/Dagster/accept.txt [hunk 3]
```diff
@@ -1,72 +1,81 @@
-[Cc]onfig
-\bDagster\b
+ACR
```
Comment (cmpadden) [inline docs/vale/styles/config/vocabularies/Dagster/accept.txt:1]: Alphabetized and moved acronyms to separate paragraph.

I originally thought we could remove these acronyms as they are in `docs/vale/styles/Dagster/acronyms.yml` but "term" errors are thrown if not present in this file.

**Comment by cmpadden** [comment]
Down to a much smaller subset:
```
$ yarn vale

 docs/guides/tbd/running-local-ui-development.md
 23:80  error  Is 'my_module' spelled          Dagster.spelling
               correctly?
 23:80  error  Did you really mean             Vale.Spelling
               'my_module'?
 53:18  error  Is 'dev' spelled correctly?     Dagster.spelling
 53:18  error  Did you really mean 'dev'?      Vale.Spelling


 docs/guides/deployment/kubernetes.md
 194:1  error  Remove or comment out TODOs.  Dagster.todos


 docs/guides/data-modeling/metadata.md
 101:99  error  Use 'S3' instead of 's3'.  Vale.Terms


 docs/dagster-plus/access/rbac/audit-logs.md
 19:48  error  Use 'RBAC' instead of 'rbac'.  Vale.Terms


 docs/dagster-plus/access/rbac/users.md
 24:19  suggestion  Replace with **Note**:          Dagster.wordiness
                    [sentence].
 45:1   error       Remove or comment out TODOs.    Dagster.todos
 51:1   error       Remove or comment out TODOs.    Dagster.todos


 docs/dagster-plus/access/authentication/pingone-sso.md
 69:10  warning  Use 'email' instead of          Terms.words
                 'Email'.
 69:34  warning  Use 'email' instead of          Terms.words
                 'Email'.


 docs/dagster-plus/access/authentication/google-workspace-sso.md
 69:32  warning  Use 'email' instead of          Terms.words
                 'Email'.


 docs/dagster-plus/saved-views.md
 68:27  suggestion  Replace with **Note**:          Dagster.wordiness
                    [sentence].


 docs/dagster-plus/insights.md
 33:14  warning  Use 'preceding' instead of      Terms.words
                 'above'.


 docs/dagster-plus/deployment/serverless/runtime-environment.md
 21:11   error    Use 'GitHub' instead of         Vale.Terms
                  'github'.
 22:79   error    Use 'GitHub' instead of         Vale.Terms
                  'github'.
 22:86   warning  Use 'Python' instead of         Terms.engineering
                  'python'.
 33:90   warning  Use 'Python' instead of         Terms.engineering
                  'python'.
 37:104  warning  Use 'Python' instead of         Terms.engineering
                  'python'.


 docs/dagster-plus/deployment/hybrid/agents/kubernetes.md
 120:5  error  Remove or comment out TODOs.  Dagster.todos

✖ 12 errors, 7 warnings and 2 suggestions in 154 files.
```

File: docs/.vale.ini [hunk 4]
```diff
@@ -26,11 +26,33 @@ mdx = md
...
+; because when these wrap markdown it causes the markdown linting to fail. For example, code blocks
+; within a tab item.
+
+TokenIgnores = (`[^`]*`), \
+    (\[.*\]\([^)]+\)), \
+    (\\\{[^}]+\}), \
+    (<\/?TabItem.*>), \
+    (<\/?Tabs.*>), \
+    (</?details>), \
+    (<summary>.*<\/summary>)
```
Comment (cmpadden) [inline docs/.vale.ini:58]: @erinkcochran87 / @PedramNavid this should run green now.

Markdown wrapped in `<Tabs>` or `<details>` was not being properly linted (for example, ignoring code blocks).

By including this HTML in the `TokenIgnores`, it is stripped from the document before linting, leaving us with the markdown contents.

This feels very finicky, and I hope we can find something more elegant in the future.

**Review by PedramNavid — APPROVED** [review]

**Pull request merged**